homeassistant/components/zwave/node_entity.py

```diff
@@ -201,17 +196,17 @@ def state(self):
         """Return the state."""
         if ATTR_READY not in self._attributes:
             return None
-        stage = ''
-        if not self._attributes[ATTR_READY]:
-            # If node is not ready use stage as sub-status.
-            stage = self._attributes[ATTR_QUERY_STAGE]
+
         if self._attributes[ATTR_FAILED]:
-            return sub_status('Dead', stage)
+            return 'dead'
+        if self._attributes[ATTR_QUERY_STAGE] != 'Complete':
```

Comment:
I wonder what should take precedence - 'Sleeping' or 'Initializing'.
If Sleeping takes precedence - a device will be marked as Sleeping fast and will be briefly initializing on first wakeup.
If Initializing takes precedence - the device would be "Initializing" for a long time until it wakes up.

I don't rreally have a preference

Reply:
Yeah, I personally think it will be more useful to know that sleeping does in fact mean that it's been seen once, but went back to sleep. That way you know that ready/sleeping means the device is working.